src/css-utils.js

```diff
@@ -90,34 +47,25 @@ CSSParserUtils.SEMICOLON                                 = -2;
 * parsing in the downstream. Please be noted that &gt; itself will cause parsing error
 * no matter decode or not if it does not belong to the STRING token.
 *
-* These function decode more than we need to prevent CSS parsing error.
+* These function decode did slightly more than html decoding, we replace \r\n\f\t with space for
+* secure-handlebars for filter judgement without considering those chars.
 *
 * Reference:
 * http://www.w3.org/TR/css-style-attr/
 *
 */
-// TODO: use client side for html decode in dist/*.js version
-CSSParserUtils.reCharReferenceDecode = /&([a-z]{2,};?)|&#0*(x[a-f0-9]+|[0-9]+);?/ig;
+CSSParserUtils.reHtmlDecode = /(\t|\r|\n|\f)/g;
+CSSParserUtils.HtmlDecodeReplacement = {
+    '\t'         : ' ', // 0x09
+    '\r'         : ' ', // 0x0A
+    '\n'         : ' ', // 0x0C
+    '\f'         : ' ', // 0x0D
+};
```

Comment:
I see you're now using a full htmldecoder. Please factor these to avoid ambiguities.